test cases/common/236 external project/libfoo/libfoo.c

```diff
@@ -0,0 +1,8 @@
+#include "libfoo.h"
+
+int func(void);
```

Comment:
Maybe also add a `func.h` instead of forward declaring to test that include paths work?

Reply:
right, done.